lib/pages/emergency_dashboard.dart

```diff
@@ -69,7 +70,7 @@ class _HomState extends State<Hom> with SingleTickerProviderStateMixin {
                   radius: 1.5)),
           child: Padding(
             padding: const EdgeInsets.all(8.0),
-            child: Column(
+            child: ListView(
               // mainAxisAlignment = MainAxisAlignment.spaceEvenly
```

Comment:
add shrinkwrap as true here is listview so that the ListView only occupies the space it needs